types/wavesurfer.js/src/mediaelement.d.ts

```diff
@@ -0,0 +1,49 @@
+import { Peaks, WebAudioBackend } from "../types/backend";
+import { WaveSurferParams } from "../types/params";
+
+export default class MediaElement extends WebAudioBackend {
+    constructor(params: WaveSurferParams);
+
+    /** Create a timer to provide a more precise audioprocess event. */
+    createTimer(): void;
+    /** This is called when wavesurfer is destroyed. */
+    destroy(): void;
+    /** Returns the current time in seconds relative to the audio-clip's duration. */
+    getCurrentTime(): number;
```

Comment:
WaveSurfer comments are wrong again and this can return `undefined` if the media isn't initialized.

https://github.com/katspaugh/wavesurfer.js/blob/11f31aeeab7f7d4e244530a62b4e973e8509dac1/src/mediaelement.js#L241

@thijstriemstra I guess we should change that code to return 0 if media is null, then the return type can always be `number`

Reply:
Fixed in be2d8d6.